x-pack/plugins/index_management/public/application/sections/home/enrich_policies_list/empty_states/empty_state.tsx

```diff
@@ -29,12 +33,32 @@ export const EmptyState = () => {
         </h2>
       }
       body={
-        <p>
-          <FormattedMessage
-            id="xpack.idxMgmt.enrichPolicies.list.emptyPromptDescription"
-            defaultMessage="Use an enrich policy to add data from existing indices into incoming documents during ingest."
-          />
-        </p>
+        <>
+          <p>
+            <FormattedMessage
+              id="xpack.idxMgmt.enrichPolicies.list.emptyPromptDescription"
+              defaultMessage="Use an enrich policy as a lookup reference, to add fields to incoming documents during ingest with {ingestPipelinesLink} or to use the ENRICH command in ES|QL"
+              values={{
+                ingestPipelinesLink: (
+                  <EuiLink
+                    href={share.url.locators.get('INGEST_PIPELINES_APP_LOCATOR')?.useUrl({})}
+                  >
+                    <FormattedMessage
+                      id="xpack.idxMgmt.enrichPolicies.list.emptyPromptIngestPipelinesLink"
+                      defaultMessage="ingest pipelines"
+                    />
+                  </EuiLink>
+                ),
+              }}
+            />
+          </p>
+          <EuiLink href={documentationService.getCreateEnrichPolicyLink()} target="_blank">
```

Comment:
@ElenaStoeva this link should go to https://www.elastic.co/guide/en/elasticsearch/reference/master/ingest-enriching-data.html as @linyaru [mentioned](https://www.figma.com/file/eXeAGnXIurjZsU0VDDmfyD?type=design&node-id=17-58926&mode=design#747549265) in the here into the Figma file

Reply:
I updated the link, thanks!